utility/utils.py

```diff
@@ -594,6 +594,8 @@ def set_config_param(node):
     # restart osd service
     node.exec_command(cmd=f"ceph orch restart {osd_process_name}")
     node.exec_command(cmd="ceph config dump")
+    log.info("sleeping for 20 seconds")
+    time.sleep(20)
```

Comment:
Blind sleep may work today but it could fail later or in the next build.

IMHO, we could check if the rgw service is up and running using
```
ceph orch ls --service_type rgw --service-name <rgw_service_name> --format json |
jq '.[0].status | select(.size !=0) | .size == .running`
```

Reply:
I might be wrong with the need for checking rgw based on the heading... however the code is restarting osd_process_name

It is also possible to check for that particular daemon running status.